instrumentation/mongo/test/test_helper.rb

```diff
@@ -0,0 +1,54 @@
+# frozen_string_literal: true
+
+# Copyright 2019 OpenTelemetry Authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+require 'mongo'
+
+require 'opentelemetry/sdk'
+
+require 'minitest/autorun'
+require 'rspec/mocks/minitest_integration'
+require 'webmock/minitest'
+
+# global opentelemetry-sdk setup:
+EXPORTER = OpenTelemetry::SDK::Trace::Export::InMemorySpanExporter.new
+span_processor = OpenTelemetry::SDK::Trace::Export::SimpleSpanProcessor.new(EXPORTER)
+
+OpenTelemetry::SDK.configure do |c|
+  c.add_span_processor span_processor
+end
+
+class Module
+  include Minitest::Spec::DSL
+end
+
+module TestHelper
+  extend self
+
+  def setup_mongo
+    Mongo::Logger.logger.level = ::Logger::WARN
+    client
+  end
+
+  def teardown_mongo
+    client.database.drop
+  end
+
+  def client
+    @client ||= Mongo::Client.new(["#{host}:#{port}"], database: database)
+  end
+
+  def database
+    'otel_test'
+  end
+
+  def host
+    ENV.fetch('TEST_MONGODB_HOST', '127.0.0.1')
```

Comment:
I believe the datadog test approach relies on a separate mongodb container in the docker network. I think we'd need to add this env var pointing to the appropriate container in our docker-compose.yml https://github.com/open-telemetry/opentelemetry-ruby/blob/master/docker-compose.yml and add both the mongodb container and the relevant env vars to the ci.yml https://github.com/open-telemetry/opentelemetry-ruby/blob/master/.github/workflows/ci.yml

Reply:
The CI passes fine here as-is. If you want to run this on a different CI, you will need to set `TEST_MONGODB_HOST` and `TEST_MONGODB_PORT` in that CI's runner script. I'm marking this as resolved as I don't see any code change is needed.